components/breadcrumb.tsx

```diff
@@ -13,7 +14,13 @@ export function BreadCrumb({ pathParts }: IBreadCrumbProps) {
       {pathParts.slice(0, -1).map((pathPart, index) => (
         <React.Fragment key={index}>
           <H6>
-            <a onClick={pathPart.onClick}>{pathPart.label}</a>
+            {!!pathPart.to ? (
+              <Link onClick={pathPart.onClick} to={pathPart.to}>
+                {pathPart.label}
+              </Link>
+            ) : (
+              <a onClick={pathPart.onClick}>{pathPart.label}</a>
```

Comment:
do you need this at all? looks like you could use link for both cases

Reply:
Yup nice